python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -69,7 +72,7 @@ class MedianAction(ScalarFromVectorAction):
     def __call__(self, data: KeyedData, **kwargs) -> Scalar:
         mask = self.getMask(**kwargs)
         values = data[self.vectorKey.format(**kwargs)][mask]
-        med = nanMedian(values) if len(values) else np.NaN
+        med = nanMedian(values) if len(values) else np.nan
```

Comment:
I was fixing these and others on #275 so if you were to review that I could merge it ahead of you.

Reply:
Sounds good!